Frontend/research/src/App.css

```diff
@@ -1,57 +1,150 @@
 .App {
   text-align: center;
+  overflow: hidden;
+}
+
+#root{
+  background-image: url('https://images.pexels.com/photos/227396/pexels-photo-227396.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940');
+  min-height: 800px;
+  background-attachment: fixed;
 }
 
 nav.header {
-  background-color: #282c34;
+  background-color: orange;
   display: flex;
+  flex-direction: column;
   align-items: center;
   justify-content: space-between;
   font-size: calc(10px + 2vmin);
   color: white;
-  height: 65px;
+  top: 0;
```

Comment:
is top: 0 doing anything here for you? i see you commented out position:fixed, was this an artifact from that?